litezip/cli.py

```diff
@@ -21,17 +31,59 @@ def _arg_parser():
     return parser
 
 
+console_logging_config = {
```

Comment:
We need to make this level of config and logging standard for all our cli tools.